src/System.Runtime.InteropServices.RuntimeInformation/src/RuntimeInformation.cs

```diff
@@ -19,7 +20,7 @@ public static string FrameworkDescription
         {
             get
             {
-                return FrameworkName;
+                return string.Format("{0} {1}", FrameworkName, typeof(object).GetTypeInfo().Assembly.GetName().Version);
```

Comment:
Can't you get the FileVersionAttribute from the assembly using Reflection APIs?  And then use it to get the version string.

If that isn't possible, I wouldn't know off the top of my head. @stephentoub or @weshaggard may have a better idea.